tools/integration_tests.py

```diff
@@ -45,6 +45,20 @@ def integration_tests(deno_executable):
     ])
     assert len(tests) > 0
     for test_filename in tests:
+        # TODO Reenable these tests Getting this error:
+        # error TS5009: Cannot find the common subdirectory path for the input
+        # files.
```

Comment:
@kitsonk Any idea about this error? I think I remember you encountering this before.

Reply:
Hmmm... strange... it usually only comes up when you are dealing with symlinks of files.  They all seem to be related to situations the improvements around the extensionless resolution though (#1322) and I wonder if the responses back to TypeScript changed in a way where it can't "find" the real modules.  I will try to dig into it more, but I would look under `-D` as well to see if there is anything weird that TypeScript might be calling.